event.js

```diff
@@ -0,0 +1,45 @@
+var button = document.querySelector('#newsButton');
+button.addEventListener('click', doSomething);
+
+function doSomething() {
```

Comment:
Now that you've got the hang of how to use 'addEventListener', you should give your function a better name.

'doSomething' doesn't tell me as the reader what changes happen to your HTML page when the button is clicked.

Can you think of a better name which describes what the function does? :)